apps/veil/src/pages/tournament/ui/round/current-voting-results/index.tsx

```diff
@@ -0,0 +1,165 @@
+import { Card } from '@penumbra-zone/ui/Card';
+import { Text } from '@penumbra-zone/ui/Text';
+import { TableCell } from '@penumbra-zone/ui/TableCell';
+import { Density } from '@penumbra-zone/ui/Density';
+import { Pagination } from '@penumbra-zone/ui/Pagination';
+import { Skeleton } from '@/shared/ui/skeleton';
+import { ValueView, Metadata } from '@penumbra-zone/protobuf/penumbra/core/asset/v1/asset_pb';
+import { Amount } from '@penumbra-zone/protobuf/penumbra/core/num/v1/num_pb';
+import { TableRow } from './table-row';
+
+const THRESHOLD = 0.05;
+
+const valueView = new ValueView({
+  valueView: {
+    value: {
+      amount: new Amount({ lo: 133700000n }),
+      metadata: new Metadata({
+        base: 'um',
+        display: 'um',
+        denomUnits: [
+          {
+            denom: 'um',
+            exponent: 6,
+          },
+        ],
+        symbol: 'um',
+        penumbraAssetId: { inner: new Uint8Array([1]) },
+        coingeckoId: 'um',
+        images: [],
+        name: 'um',
+        description: 'um',
+      }),
+    },
+    case: 'knownAssetId',
+  },
+});
+
+export const CurrentVotingResults = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [page, setPage] = useState(1);
+  const [limit, setLimit] = useState(10);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setIsLoading(false);
+    }, 1000);
+  }, []);
+
+  const totalPages = 100;
+  const totalVotes = 10000;
+
+  const data = [
+    {
+      symbol: 'USDC',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 2777,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'OSMO',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 1337,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'ATOM',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 1000,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'USDC',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 7,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'OSMO',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 3,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'ATOM',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 10,
+      estimatedIncentive: valueView,
+    },
+  ];
+
+  return (
+    <Card>
+      <div className='flex flex-col p-3 gap-4'>
+        <Text xxl color='text.primary'>
+          Current Voting Results
+        </Text>
+        <Density compact>
+          <div className='grid grid-cols-5 h-auto overflow-auto'>
```

Comment:
issue: Tailwind's `grid-cols-{number}` expands as `grid-template-columns: repeat(<number>, minmax(0, 1fr))`, which allows shrinking of columns to fit the viewport. This leads to a table without horizontal scroll, preferable in our case. I'd change it to `grid-cols-[auto_auto_auto_auto_auto]` or `grid-cols-[1fr_1fr_1fr_1fr_1fr]`. If this is implemented along with `whitespace: no-wrap` within the rows, it'd look better for mobile screens.

<img width="293" alt="image" src="https://github.com/user-attachments/assets/74851f21-5277-4e2f-9f5b-609ee46149de" />

Reply:
Wasn't aware, thanks for the tip!